homeassistant/components/devolo_home_control/binary_sensor.py

```diff
@@ -81,13 +81,14 @@ def __init__(
             or self._binary_sensor_property.sensor_type
         )
 
-        if self._attr_device_class is None:
-            if device_instance.binary_sensor_property[element_uid].sub_type != "":
-                self._attr_name += (
-                    f" {device_instance.binary_sensor_property[element_uid].sub_type}"
-                )
-            else:
-                self._attr_name += f" {device_instance.binary_sensor_property[element_uid].sensor_type}"
+        if device_instance.binary_sensor_property[element_uid].sub_type != "":
+            self._attr_name = device_instance.binary_sensor_property[
+                element_uid
+            ].sub_type.capitalize()
```

Comment:
This is provided by the device/user configurable, right?

So if this was, for example, "Device from Frenck" it would become "Device from frenck"?

Reply:
It comes from the device, but wasn't nice in the past. In the past we stripped the sensor type if you could map it to a device class. However with the new entity naming I found it prettier to always use the the sensor type.

Reply:
Sure, but that didn't confirm nor debunk my example above. Could you clarify the behavior?

Reply:
Right, my concern is `capitalize()` in this case. If the name comes from the device, and I named it "Device of Frenck" it should not lower the case on my name, as my name is with a capital `F`.

Reply:
It doesn't, because the capitalize is only applied to the sensor type (door in this example). The sensor type is a device property that the user cannot change. The name the user gave the device stays untouched as set in devoloe_device.py#L41